Framework/BrowserSettings/ConfigReader.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Microsoft.Extensions.Configuration;
+
+namespace Framework.BrowserSettings
+{
+    class ConfigReader
+    {
+        protected static IConfiguration config;
```

Comment:
config is never used and always will be null

Reply:
Fixed. Value will be assigned when `Config ` is invoked first time.